### PR TITLE
Begin adding content to hobbies page.

### PR DESCRIPTION
**Pull request by ganewman**
Begin adding content to hobbies page. Add description, table, and links to ballroom-focused section. Add description and add placeholder headers for images to embroidery section. Finally, begin creating list of books for reading section. Likely will add additional images and styling later.

### CODE REVIEW

File: portfolio/src/main/webapp/hobbies.html [hunk 1]
```diff
@@ -9,6 +9,125 @@
   <body>
     <div id="content">
       <h1>My Hobbies</h1>
+      <h3>Here is a list of some of my many hobbies (in no particular order):
```
Comment (suparnabarua) [inline portfolio/src/main/webapp/hobbies.html:12]: Can you keep the h3 tag formatting consistent? E.g. for the h3 tag in line 12, you're adding the closing tag in line 13. Whereas in line 25, the opening and closing tag are in the same line.  It's up to you to pick the style you prefer, as long as it is consistent.

File: portfolio/src/main/webapp/hobbies.html [hunk 2]
```diff
@@ -9,6 +9,125 @@
...
+      </h3>
+      <ul>
+        <li><a href="#ballroom">Ballroom and latin dancing</a></li>
+        <li><a href="#xstitch">Cross stitching</a></li>
+        <li>Knitting</li>
```
Comment (suparnabarua) [inline portfolio/src/main/webapp/hobbies.html:17]: Will you be adding links for Knitting and the other lists as well? If so , add a TODO comment

https://google.github.io/styleguide/htmlcssguide.html#Action_Items

File: portfolio/src/main/webapp/hobbies.html [hunk 3]
```diff
@@ -9,6 +9,125 @@
...
+        but here are some images of patterns I am currently working on,
+        which have been completed by other people.
+    <h5>Pretty Little Tokyo</h5>
+    <h5>Japanese Garden</h5>
+    <h5>Gustav Klimpt's Mother and Child</h5>
+    <h3 id="reading">Reading</h3>
+    I love to read! Here are some of my favorite books:
+      <ul>
+          <li>Pachinko by Min Jin Lee</li>
+          <li>Ender's Game (series) by Orson Scott Card</li>
```
Comment (kellyktan) [inline portfolio/src/main/webapp/hobbies.html:122]: Love ender's game!

File: portfolio/src/main/webapp/hobbies.html (hunk 2 above)
Comment (ganewman) [reply portfolio/src/main/webapp/hobbies.html:17]: I wasn't necessarily planning on it, but I could if I have time. Would you still advise adding a TODO if it's a stretch goal?

File: portfolio/src/main/webapp/hobbies.html (hunk 1 above)
Comment (ganewman) [reply portfolio/src/main/webapp/hobbies.html]: Should be fixed!

File: portfolio/src/main/webapp/hobbies.html (hunk 2 above)
Comment (suparnabarua) [reply portfolio/src/main/webapp/hobbies.html:16]: In that case, don't add the TODO :)

**Pull request merged**